src/bc_protocol.rs

```diff
@@ -69,6 +71,9 @@ pub enum Error {
     #[error(display = "ADPCM Decoding Error")]
     AdpcmDecodingError(&'static str),
 
+    #[error(display = "Camera responded with Service Unavaliable")]
+    CameraServiceUnavaliable,
```

Comment:
Is there a semantic distinction between this and UnintelligibleReply?

Reply:
From my understanding:
Unintellible is a reply that wasn't understood by Neolink.
It is likely a Neolink failure in the receive direction due to incompatible device or firmware.
But could be bad comms or hardware failure.

Unavailable is the device rejected the request. Neolink understood this rejection.
It may be a failure by Neolink to send a well formatted request, possibly due to incompatible device or firmware.
But could also just be normal behaviour by the device at certain times.

Reply:
This one happens when neolink receives a valid BC packets but that the response code is not 200.

I think unintelligible is for when the response is not even deserialisable while this one is for when the camera says no.

For example when trying to start two talk streams the camera will respond with a non 200 code until the first client says it has finished talking.